### ISSUE TITLE
cannot import name gl_info 

### ISSUE THREAD

**Issue by BeibinLi** [psychopy/psychopy#1065]
My psychoPy runs fine last week, but I encountered this problem today. I cannot even run the hello world now.
I also tried to uninstall & install psychopy and Python27 today. So, I have a clean installation of Python and PsychoPy now. 

I am using *Windows 7, Python 2.7.10, and PsychoPy 1.83.01* (I also tried 1.82.01).

```python
##### Running: C:\Program Files (x86)\PsychoPy2\lib\site-packages\psychopy-1.83.01-py2.7.egg\psychopy\demos\coder\basic\hello_world.py #####
Traceback (most recent call last):
  File "C:\Program Files (x86)\PsychoPy2\lib\site-packages\psychopy-1.83.01-py2.7.egg\psychopy\demos\coder\basic\hello_world.py", line 8, in <module>
    from psychopy import visual, core
  File "C:\Program Files (x86)\PsychoPy2\lib\site-packages\psychopy-1.83.01-py2.7.egg\psychopy\visual\__init__.py", line 15, in <module>
    from psychopy.visual.window import Window, getMsPerFrame, openWindows
  File "C:\Program Files (x86)\PsychoPy2\lib\site-packages\psychopy-1.83.01-py2.7.egg\psychopy\visual\window.py", line 51, in <module>
    from psychopy import core, platform_specific, logging, prefs, monitors, event
  File "C:\Program Files (x86)\PsychoPy2\lib\site-packages\psychopy-1.83.01-py2.7.egg\psychopy\event.py", line 36, in <module>
    from pyglet.window.mouse import LEFT, MIDDLE, RIGHT  # takes ~250ms, so do it now
  File "C:\Program Files (x86)\PsychoPy2\lib\site-packages\pyglet\window\__init__.py", line 133, in <module>
    from pyglet.gl import gl_info
  File "C:\Program Files (x86)\PsychoPy2\lib\site-packages\pyglet\gl\__init__.py", line 101, in <module>
    from pyglet.gl import gl_info
ImportError: cannot import name gl_info
```

**Comment by peircej** [comment]
This isn't really a PsychoPy issue either (I notice you asked about it and @qbilius suggested you come here). If simply running the line `from pyglet import gl_info` is causing the error message then it's a problem with pyglet that I probably can't fix.

But I am keen to work out what's happening since lots of us depend on pyglet. Can you think through what changes you made since last week? Did you upgrade PsychoPy? Did you install any updates on your system?

Based on a google search:
https://www.google.co.uk/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=ImportError%3A+cannot+import+name+gl_info
I found one other psychopy user said the issue was with having unicode characters in the path of the script they were running:
https://groups.google.com/forum/#!topic/psychopy-users/sStu7k6308g

Could that be the same for you?

Basically I don't have a clue so we'll need more info
cheers, Jon

**Comment by BeibinLi** [comment]
I tried to upgrade PsychoPy 1.82 to 1.83 by running buildWinDistributions.bat, setup.py, and some other files. However, I failed to install the new version, so i downloaded and installed the "StandalonePsychoPy-1.83.01-win32.exe" directly. Then, pyglet got the problem.

I tried to reinstall Python27 and "pip install pyglet", but same error occurs for clean installation of Python and pyglet. However, the pyglet package is a correct one, because it works fine on another computer. I am afraid the bug is caused by a system error.

The problem occurs even if I just simply call "import pyglet.gl"

**Comment by jeremygray** [comment]
This post suggests that possible incompatibilities might cause this, e.g., using pyglet 1.1.4 with 64-bit python https://stackoverflow.com/questions/16308100/enthought-canopy-64bit-on-osx-import-pyglet-gl-failure

In a set-up in which you get the error, what version of pyglet do you have? And what flavor of python 2.7 (64 vs 32 bits)?

This is what I have (from within a python shell, e.g., through the Coder shell window):

    >>> import pyglet
    >>> pyglet.version
    '1.2.4' 
    >>> import sys
    >>> sys.version
    '2.7.10 |Anaconda 2.0.1 (x86_64)| (default, Oct 19 2015, 18:31:17) \n[GCC 4.2.1 (Apple Inc. build 5577)]'

Here x86_64 means its 64-bit

**Comment by BeibinLi** [comment]
Thanks Jeremy, I have read that post before. Here is my python and pyglet version. 
I guess some system settings are changed so that I cannot run pyglet any more. 

Thanks for all  the helps, and I will also post this question to pyglet team. 

```Python
Python 2.7.10 (default, May 23 2015, 09:40:32) [MSC v.1500 32 bit (Intel)] on win32
Type "copyright", "credits" or "license()" for more information.
>>> import pyglet
>>> pyglet.version
'1.2.4'   
# Note: I also tried 1.2alpha1, which is installed in the Standalone PsychoPy exe package
>>> import sys
>>> sys.version
'2.7.10 (default, May 23 2015, 09:40:32) [MSC v.1500 32 bit (Intel)]'
# Note: I also tried 2.7.3,  23 bit, which is installed in the Standalone PsychoPy exe package

>>> from pyglet.gl import Window

Traceback (most recent call last):
  File "<pyshell#5>", line 1, in <module>
    from pyglet.gl import Window
  File "C:\Python27\lib\site-packages\pyglet\gl\__init__.py", line 101, in <module>
    from pyglet.gl import gl_info
ImportError: cannot import name gl_info
```

**Comment by hoechenberger** [comment]
Closing as not a PsychoPy bug.
